### DISCUSSION TITLE
Would it be nice to have a guide on how to migrate from sqflite to drift?

### DISCUSSION THREAD

**Discussion by jari-kuipers** [simolus3/drift#3035] [Ideas]
Hi all! Thanks all around for the library!

I have to say, I'm aching to move our applications to drift. It looks really appealing. All our applications right now have sqflite implementations at the moment with custom code for migrations etc.

Bottom line, all of our users already have a .db file somewhere in their storage.

Right now I'm struggling to move from sqflite to drift without breaking our existing users' precious data and I was thinking with the website having a "Documentation & Guides" part if it would be a good idea to include a migration guide there at some point like the Flutter docs themselves or Riverpod for example.

**Comment by simolus3** [comment]
Thanks for the suggestion! I agree that having some "official" pointers on how to migrate existing databases is useful. I have started writing that down [here](https://666df24e7b5d7f223325594f--moor.netlify.app/docs/migrating_to_drift/), is that roughly what you were looking for?

**Comment by dickermoshe** [comment]
Part of the docs rewrite will have guides for each of the popular db libs
